PsfLauncher/main.cpp

```diff
@@ -101,6 +147,105 @@ catch (...)
     return win32_from_caught_exception();
 }
 
+bool LaunchViaCreateProcessInContainer(std::wstring app, std::wstring fullCmd, std::wstring dir, bool cmdShow)

```

Comment:
There is already a fix in PSF that I added last week to make sure Powershell.exe is always launched inside the container.  In StartProcessHelper.h there is a method called MakeAttributeList. This modifies the breakaway behavior.

I think you were seeing an issue that I recently fixed where the breakaway behavior was not correct in release builds. That issue has been fixed.  Please bring in all changes from develop to see them.